src/pages/docs/packaging-applications/build-servers/github-actions.md

```diff
@@ -109,10 +108,9 @@ jobs:
 | `release_number`      | The number for the new Release. If omitted, Octopus Deploy will generate a Release number.                                                           |
 | `environments`        | A list of Environments in Octopus Deploy in which to run (i.e. Dev, Test, Prod). Add each environment on a new line.                                 |
 | `variables`           | A list of Variables to use in the Deployment in `key: value` format. Add each variable on a new line.                                                |
-| `git_ref`             | The Git branch from which to source the project code. Required for Projects using version control in Octopus.                                        |
-| `git_commit`          | The Git commit from which to source the project code. Required for Projects using version control in Octopus.                                        |
-| `server`              | The base URL hosting Octopus Deploy (i.e. `https://octopus.example.app`). It is strongly recommended that this value retrieved from a [GitHub secret](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions). |
```

Comment:
The `server` input looks like it is still actually used in the example, in the `OctopusDeploy/login` step, I wonder if it might be worth keeping it in here in addition to the env var section.